### PR TITLE
Update the repository version in the Space ROS Dockerfiles

### PR DESCRIPTION
**Pull request by mjeronimo**
Adapt to changes in various repos, bringing things up to date.
The MoveGroup demo has also moved from the MoveIt2 repository
to the MoveIt2 tutorials repository, so also update the READMEs
accordingly as well.

Signed-off-by: Michael Jeronimo <michael.jeronimo@openrobotics.org>

### CODE REVIEW

File: moveit2/README.md [hunk 1]
```diff
@@ -105,56 +105,4 @@ You can also test the graphics output by running 'glmark2'. A window like the fo
...
-```
-
-The test results will be in the test output directory, /root/src/moveit2_ws/build/moveit_ros_move_group/test_results/.
-
-## Open Issues
-
-* With the latest MoveIt2 source, compiled against the latest Space ROS source code, there is currently a compile-time error in realtime_server_goal_handle.h. There is a patch installed (patch-1.diff) in the Dockerfile to work around this.
-
-* There are two versions of libcurl installed on the system, one installed along with the curl utility (installed in the base Space ROS image) and another that comes along with other installed libraries. There are warnings output when running utilities, such as cmake, that load libcurl. However, this seems not to affect the build process.
+Once GUIs and graphics are verified to be working from the container, you can move on to the next step: [building and running MoveIt2 Tutorials](../moveit2_tutorials/README.md).
```
Comment (nuclearsandwich) [inline moveit2/README.md:108]: Was dropping all the content after this initial sentence intentional?

**Review by nuclearsandwich — COMMENTED** [review]
Overall looking good. One fairly significant question raised regarding documentation which I think needs addressing before approval.

File: moveit2/README.md (hunk 1 above)
Comment (mjeronimo) [reply moveit2/README.md:108]: @nuclearsandwich  Yes, the maintainers of the MoveIt2 repository have moved out some of the sample apps into the moveit2_tutorials repo. In particular, the move_group demo that I was using before has been eliminated. So, this section is now obsolete. In the moveit2_tutorials README, I've added a different test to run and have pointed to the online documentation for the details of running the test.

Comment (nuclearsandwich) [reply moveit2/README.md:108]: Thanks for elaborating!

**Review by nuclearsandwich — APPROVED** [review]

**Pull request merged**